test/performance/odp_l2fwd.c

```diff
@@ -2288,6 +2285,11 @@ int main(int argc, char *argv[])
 	gbl_args->pkt_len = pkt_len;
 	gbl_args->seg_len = seg_len;
 
+	/* Print resulting values after option values */
+	printf("Packets per pool:   %u\n", num_pkt);
```

Comment:
Could print some heading first. Based on only the application output it's not that clear why some option values are printed twice. All resulting value prints could also be combined into a function.

Reply:
Added heading print in v4. Prints are done before normal and vectors pool create calls. Those are in two places and didn't want to refactor pool create code paths.